sound/soc/sof/imx/imx-common.c

```diff
@@ -60,6 +60,13 @@ void imx8_dump(struct snd_sof_dev *sdev, u32 flags)
 	/* Get information about the panic status from the debug box area.
 	 * Compute the trace point based on the status.
 	 */
+	if (sizeof(status) + 4 > sdev->debug_box.size) {
```

Comment:
@plbossart I understand, that it's the same 4 as below when copying from `offset + 4`. So, yes, it would be good to clarify both

Reply:
This is IMX specific offset for the panic code. @dbaluta ? Not sure if dev_err() is correct. If DSP has panic'ed and debug_box is for some reason not available, it would seem you should just skip the debug dump in this case. E.g. for this case, dumping the registers would still be useful even if status is not avalable.